libopenage/datastructure/constexpr_map.h

```diff
@@ -268,15 +238,13 @@ class ConstMap<K, V, 0> {
 	 * Fetching a value from the end element aborts as it has no entry.
 	 */
 	constexpr const inline V &fetch(const K &) const {
-		return this->missing_key();
+		// A key is not in the map.
+		// This function should never be called. If it is called,
+		// you requested a nonexisting key.
+		return *static_cast<const V*>(nullptr);
```

Comment:
That won't work either, because the template is instantiated on use, unless you work out some type-level `get` that only instantiates `fetch` with the key that we're trying to find.. But it's pointless unless you're interested in template hackery, and I realized a constexpr map is not really what we need, since I use it at runtime anyway, so I think I'll revert to using switch statements.